.pipelines/templates/.builder-release-template.yaml

```diff
@@ -22,12 +22,12 @@ steps:
     displayName: Get Build Mode
 
   - task: UniversalPackages@0
-    displayName: Download Asc Baseline
```

Comment:
this serves as an ASC baseline replacement?

Reply:
Yes it does, the outdated ASC baseline binaries that we consumed here set settings that overlap and are incompatible with CIS.